api/admin/controller/settings.py

```diff
@@ -407,45 +406,14 @@ def _get_prior_test_results(self, item, protocol_class=None, *extra_args):
         self_test_results = None
 
         try:
-            if self.type == "collection":
-                if not item.protocol or not len(item.protocol):
-                    return None
-
-                if not protocol_class:
-                    registry = LicenseProvidersRegistry()
-                    protocol_class = registry.get(item.protocol)
-
-                if item.protocol == OPDSImportMonitor.PROTOCOL:
-                    protocol_class = OPDSImportMonitor
-                    extra_args = (OPDSImporter,)
-
-                if issubclass(protocol_class, BaseHasSelfTests):
-                    self_test_results = protocol_class.prior_test_results(
-                        self._db, protocol_class, self._db, item, *extra_args
-                    )
-
-            elif self.type == "search service":
+            if self.type == "search service":
```

Comment:
So I removed them here, since they were no longer getting tested, and it was causing out code coverage to go down, but these are only removed here since we are no longer using the settings controller for collection tests. The collection tests run through `IntegrationSelfTestsController`, which still handles them.